README.md

```diff
@@ -33,70 +33,10 @@ If you want to set up a Virtual Schema for a different database system, please h
-| [Maven Compiler Plugin][maven-compiler-plugin]                     | Setting required Java version                          | Apache License 2.0            |
-| [Maven Assembly Plugin][maven-assembly-plugin]                     | Creating JAR                                           | Apache License 2.0            |
-| [Versions Maven Plugin][versions-maven-plugin]                     | Checking if dependencies updates are available         | Apache License 2.0            |
-| [Maven Enforcer Plugin][maven-enforcer-plugin]                     | Controlling environment constants                      | Apache License 2.0            |
-| [Maven Dependency Plugin][maven-dependency-plugin]                 | Accessing to test dependencies                         | Apache License 2.0            |
-| [Artifact Reference Checker Plugin][artifact-ref-checker-plugin]   | Check if artifact is referenced with correct version   | MIT License                   |
-| [Project Keeper Maven Plugin][project-keeper-maven-plugin]         | Checking project structure                             | MIT License                   |
-| [Error Code Crawler Plugin][error-code-crawler-plugin]             | Analyzing used error messages                          | MIT License                   |
-| [Sonatype OSS Index Maven Plugin][sonatype-oss-index-maven-plugin] | Checking dependencies vulnerability                    | ASL2                          |
-
-[virtual-schema-common-jdbc]: https://github.com/exasol/virtual-schema-common-jdbc
-[db2-jdbc-driver]: https://www.ibm.com/support/pages/node/382667
-[exasol-error-reporting]: https://github.com/exasol/error-reporting-java/
-
-[exasol-hamcrest]: https://github.com/exasol/hamcrest-resultset-matcher
-[test-db-builder]: https://github.com/exasol/test-db-builder-java
-
-[maven-jacoco-plugin]: https://www.eclemma.org/jacoco/trunk/doc/maven.html
-[maven-surefire-plugin]: https://maven.apache.org/surefire/maven-surefire-plugin/
-[maven-compiler-plugin]: https://maven.apache.org/plugins/maven-compiler-plugin/
-[maven-assembly-plugin]: https://maven.apache.org/plugins/maven-assembly-plugin/
-[versions-maven-plugin]: https://www.mojohaus.org/versions-maven-plugin/
-[maven-enforcer-plugin]: http://maven.apache.org/enforcer/maven-enforcer-plugin/
-[artifact-ref-checker-plugin]: https://github.com/exasol/artifact-reference-checker-maven-plugin
-[maven-dependency-plugin]: https://maven.apache.org/plugins/maven-dependency-plugin/
-[project-keeper-maven-plugin]: https://github.com/exasol/project-keeper-maven-plugin
-[error-code-crawler-plugin]: https://github.com/exasol/error-code-crawler-maven-plugin
-[sonatype-oss-index-maven-plugin]: https://sonatype.github.io/ossindex-maven/maven-plugin/
-
-[virtual-schemas-user-guide]: https://docs.exasol.com/database_concepts/virtual_schemas.htm
```

Comment:
VS schemas user guide, vs-doc, and vs-api links should stay.

Reply:
Oh, you're right!